RWKV-v4neo/config-example.yaml

```diff
@@ -198,12 +201,21 @@ model:
   # Learning rate of the training process
   lr_init: 1.0e-04
 
-  # various other settings you probably should leave alone
-  grad_cp: true
-  warmup_steps: -1
+  # Adam optimizer settings
+  # You probably want to leave this alone, unless you know what you are doing
   beta1: 0.9
   beta2: 0.99
   adam_eps: 1.0e-08
+  weight_decay: 0.01
+
+  # torch.set_float32_matmul_precision, used to optimize operations with tensor cores
+  # this should be set as null, for non cuda core GPUs
+  torch_set_float32_matmul_precision: 'high'
```

Comment:
Do we really have the need for TF32 precision? AFAIK Bo is using BF16 for all matmuls.

Reply:
Wasn't sure if this would affect hidden state, which is fp32 - but can ping Bo to confirm this

Reply:
saw that you updated and merged - thanks =)